src/cli/fo_nomos_license_list.php

```diff
@@ -126,7 +126,7 @@ function GetLicenseList($uploadtree_pk, $upload_pk, $showContainer, $excluding,
   $licenseDao = $GLOBALS['container']->get("dao.license");
 
   if (empty($uploadtree_pk)) {
-      $uploadtreeRec = $dbManager->getSingleRow('SELECT uploadtree_pk FROM uploadtree WHERE parent IS NULL AND upload_fk=$1',
+      $uploadtreeRec = $dbManager->getSingleRow('SELECT uploadtree_pk FROM uploadtree WHERE parent IS NULL AND upload_fk=$2',
```

Comment:
This prepared statement only gets one argument (`$upload_pk` in `array($upload_pk)`) and thus only `$1` is defined and `$2` would point to the second one, which is not present.

Reply:
Ah, I see, any suggestions on how to fix this particular problem?